ACMILabs/playlist-label
# #101 Various Up Next template issues

Resolves #101

Updated Up Next template UI issues.

### Acceptance Criteria
- [x] Up next now shows correct time left for each playlist item
- [x] 'ss' and '0 minute' UI bugs resolved

### Relevant design files
* None

### Testing instructions
1. For Stories we tell, see that UI issues have been resolved, and the correct up next time displays.
```
LABEL_TEMPLATE=up_next.html
XOS_PLAYLIST_ID=246
XOS_MEDIA_PLAYER_ID=45
```
2. See that for Now you see me, Countdown still behaves as expected.
```
LABEL_TEMPLATE=countdown.html
XOS_PLAYLIST_ID=141
XOS_MEDIA_PLAYER_ID=76
```
